pkg/blockstorage/vmware/conversion.go

```diff
@@ -17,7 +17,7 @@ func convertFromObjectToVolume(vso *types.VStorageObject) (*blockstorage.Volume,
 		Type:         blockstorage.TypeFCD,
 		ID:           vso.Config.Id.Id,
 		CreationTime: blockstorage.TimeStamp(vso.Config.CreateTime),
-		Size:         vso.Config.CapacityInMB / 1024,
+		SizeInBytes:  vso.Config.CapacityInMB * blockstorage.BytesInM,
```

Comment:
I think this is wrong. VMware actually returns MiB even though it calls it MB.

You should instrument and verify what it returns.